cfme/scripting/appliance.py

```diff
@@ -48,6 +53,7 @@ def upgrade_appliance(appliance_ip, cfme_only, update_to):
     else:
         print('Fetching appliance from env.local.yaml')
     app = get_appliance(appliance_ip)
+    assert app.version > '5.7', "{} is not supported, must be 5.7 or higher".format(app.version)
```

Comment:
**SUGESTION:** This too could go into a config, and should not be hard coded.   Also, I would even argue that the Appliance object itself should know what its minimally supported version is.

Reply:
Again suggestions exactly what you mean would help, but basically its currently set to 5.7 as that had a major postgres upgrade, older versions need a lot more TLC to be coded, just easier to have a cut off currently. (May change in the future) In terms of the 'Appliance object' knowing its supported versions, this can again change depending on the builds and what _we_ want to actually support/implement, I have just added things that people always request my help with.

Reply:
Not objecting to a cut off, but suggesting there be some minimimal_cfme_version in the cfme_data.yaml.    It would be best if the Appliance class itself had its own minimal_supported_version() class method that read this value from the config...but just having the minimally supported CFME version in the config rather than the code is much better.

Reply:
This version specificity should be made in the framework and not in the yaml. If it is in the yaml, it means someone can alter the "supported" version. We don't want that. Again, until we have that this specified in the Appliance object, we can do this work later, but it is a very good suggestion.